packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -419,7 +419,7 @@ export class MessageController extends TransportController {
         return message;
     }
 
-    private async validateMessageRecipients(recipients: CoreAddress[]) {
+    public async validateMessageRecipients(recipients: CoreAddress[]): Promise<CoreError | undefined> {
```

Comment:
returning an error from a public api is bad practice - we have results for that.